.github/actions/get-release-issue-v1/src/run.test.ts

```diff
@@ -0,0 +1,310 @@
+  let info: sinon.SinonSpy
+  let getExecOutputStub: sinon.SinonStub
+
+  beforeEach(() => {
+    getInput = sinon.stub()
+    setFailed = sinon.spy()
+    setOutput = sinon.spy()
+    info = sinon.spy()
+    getExecOutputStub = sinon.stub(exec, 'getExecOutput')
+  })
+
+  afterEach(sinon.restore)
+
+  const getReleaseIssueRun = async (ownerAndRepo = '') => {
+    const issue = {
+      url: 'https://github.com/dequelabs/axe-api-team-public/issues/1',
+      title: `${ownerAndRepo || 'dequelabs/axe-api-team-public'} v1.0.0`
+    } as const
+
+    getInput.withArgs('version', { required: true }).returns('1.0.0')
+    getInput.withArgs('owner-and-repo').returns(ownerAndRepo)
+    getExecOutputStub.returns({
+      stdout: JSON.stringify([
+        {
+          url: 'https://github.com/dequelabs/axe-api-team-public/issues/1',
+          title: `${ownerAndRepo || 'dequelabs/axe-api-team-public'} v1.0.0`
+        }
+      ])
+    })
+
+    const core = {
+      getInput,
+      setFailed,
+      setOutput,
+      info
+    }
+
+    const github = {
+      context: {
+        repo: {
+          owner: 'dequelabs',
+          repo: 'axe-api-team-public'
+        }
+      }
+    }
+
+    await run(core as unknown as Core, github as unknown as GitHub)
+
+    return { core, issue }
+  }
+
+  describe('given `version` input is not provided', () => {
+    it('throws an error', async () => {
+      getInput.withArgs('version', { required: true }).throws({
+        message: 'Input required and not supplied: version'
+      })
+
+      const core = {
+        getInput,
+        setFailed
+      }
+
+      const github = {
+        context: {
+          repo: {
+            owner: 'dequelabs',
+            repo: 'axe-api-team-public'
+          }
+        }
+      }
+
+      await run(core as unknown as Core, github as unknown as GitHub)
+
+      assert.isTrue(core.setFailed.calledOnce)
+      assert.isTrue(
+        core.setFailed.calledWith('Input required and not supplied: version')
+      )
+    })
+  })
+
+  describe('given `owner-and-repo` input is not provided', () => {
+    it('uses the `owner` and `repo` from the github context and sets the output', async () => {
+      const { core, issue } = await getReleaseIssueRun()
+
+      assert.isTrue(
+        core.info.firstCall.calledWith(
+          'Getting issues for dequelabs/axe-api-team-public v1.0.0...'
+        )
+      )
+
+      assert.isTrue(getExecOutputStub.calledOnce)
+      assert.isTrue(
+        getExecOutputStub.calledWith(
+          'gh issue list --repo dequelabs/axe-api-team-public --label release --state open --json url,title'
+        )
+      )
+
+      assert.isTrue(core.info.calledTwice)
+      assert.isTrue(
+        core.info.secondCall.calledWith(
+          `Found issue: ${issue.url}. Setting "issue-url" output...`
+        )
+      )
+
+      assert.isTrue(setOutput.calledOnce)
+      assert.isTrue(setOutput.calledWith('issue-url', issue.url))
+    })
```

Comment:
This is testing multiple things and ensuring that each portion of the code is called in a single test. Please split this type of test out and use [single use case testing](https://github.com/dequelabs/axe-api-team/blob/main/policies/unit-testing.md#single-use-case-testing). Other tests also test multiple outputs in a single test and should be refactored.